ecr-login/api/factory.go

```diff
@@ -63,7 +63,6 @@ func (defaultClientFactory DefaultClientFactory) NewClientWithFipsEndpoint(regio
 	awsSession := session.New()
 	awsSession.Handlers.Build.PushBackNamed(userAgentHandler)
 
-	// UnknownServiceError is expected for "ecr-fips", but resolver should still generate correct FIPs endpoint
 	endpoint, _ := getServiceEndpoint("ecr-fips", region)
```

Comment:
Probably. I added it in https://github.com/aws/amazon-ecs-cli/pull/692/files#diff-b02c781577f1c080ca86c9a216863136R73, but it does make the client construtors inconsistent in their return values. Do you have a recommendation for how to handle?

Reply:
Thanks.  I'm fine with this either way; I think that we'll likely need to add `error` to the return signature at some point to allow us to move to `session.NewSession` instead of `session.New` (and considering the current released version is 0.2.0, I'm not terribly worried about a breaking change).